frontend/website/src/Style.re

```diff
@@ -1,21 +1,27 @@
 module Colors = {
   let fadedBlue = `rgb((111, 167, 197));
+  let white = `rgb((255, 255, 255));
```

Comment:
Does Css not already expose this or do we want to refine it? I might be worried someone would use the wrong one